### ISSUE TITLE
Google not working

### ISSUE THREAD

**Issue by nhaouari** [HEmile/obsidian-search-on-internet#5]
Hey, 
Thank you for the plugin, it works like a charm for other websites but for google I get the following error: 
![image](https://user-images.githubusercontent.com/9850722/106298310-07a59880-6254-11eb-81da-8670294c4c17.png)

Best regards.

**Comment by HEmile** [comment]
Hi! Are you using the default query for Google or have you modified it?

**Comment by nhaouari** [comment]
> Hi! Are you using the default query for Google or have you modified it?

I use the default query for google.

**Comment by HEmile** [comment]
Hi! Is this still an issue :)?

**Comment by nhaouari** [comment]
I have updated Obsidian to version 0.11.1, unfortunately, there is another error: 
```
webhp:130 Blocked autofocusing on a <input> element in a cross-origin subframe.
```

![image](https://user-images.githubusercontent.com/9850722/108187295-98aeb780-710e-11eb-9973-818cdc861ee7.png)

**Comment by HEmile** [comment]
Hi, you don't seem to be using the correct query for Google. It should be `https://www.google.com/search?&q={{query}}`, not `https://www.google.com/search{{query}}`

**Comment by nhaouari** [comment]
> https://www.google.com/search?&q={{query}}

I have tried many alternatives, with google, it always shows a page that looks like this:
![image](https://user-images.githubusercontent.com/9850722/108192347-6ef88f00-7114-11eb-8a89-108fcd8048ed.png)

**Comment by phibr0** [comment]
same error for me

**Comment by kinoc** [comment]
I had a problem with patents.google.com, where it seemed to execute the query, but "{{query}}" was displayed in the search box. 

> https://patents.google.com/?q={{query}}&oq={{query}}

I changed it to the following and it seem to both execute the query and display it in the search box.

> https://patents.google.com/?q={{query}}&oq={{title}}

Maybe it was only doing one replace for each substitution, so it only did one for each possibility (limit one per string).  The regular search 

> https://www.google.com/search?&q={{query}}

appears to work for me.

**Comment by HEmile** [comment]
@kinoc Indeed, that's a bug on my part: It only replaces {{query}}  a single time and {{title}} a single time... So I'm impressed you found that workaround...! I didn't realize this would be something common.

@nhaouari @phibr0 The error you're getting is the cookie screen of Google. I think you might be able to work around this by accepting the cookies on another Google site, like calendar. Otherwise, you'll have to do with eg DuckDuckGo
